spec.bs

```diff
@@ -3478,6 +3565,12 @@ An interest group is a [=struct=] with the following [=struct/items=]:
 
 </dl>
 
+A <dfn>regular interest group</dfn> is an [=interest group=] whose
+additional bid key (TODO: add link once the PR defines it is merged) is null.

```

Comment:
Does this PR exist? If so, can you add a link to it here so we can more easily follow-up on the progress of that and therefore this TODO in the future? And if it doesn't exist yet, I'd suggest filing a follow-up issue for it to track. Again I'm just trying to make the many TODOs of this spec more easily linkable and trackable, so that we can follow-up with more context in the future without having to do lots of code archaeology.

Reply:
yes, it's in the [negative targeting PR](https://github.com/WICG/turtledove/pull/796/files#diff-6f5a1d8263b0b0c42e2716ba5750e3652e359532647ac934c1c70086ae3ceddaR3982) you've been reviewing. Actually that PR also needs the two defintions, so moved the definitions [there](https://github.com/WICG/turtledove/pull/796/files#diff-6f5a1d8263b0b0c42e2716ba5750e3652e359532647ac934c1c70086ae3ceddaR4009). Will remove one of them based on which is merged first.

Reply:
Perfect, thanks!

Reply:
Done.